user_threshold/models/res_users.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+import os
+from lxml import etree
+
+from odoo import _, api, fields, models
+from odoo.exceptions import AccessError, ValidationError
+
+from .ir_config_parameter import HIDE_THRESHOLD, MAX_DB_USER_PARAM
+from .res_groups import THRESHOLD_MANAGER
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    threshold_exempt = fields.Boolean(
+        'Exempt User From User Count Thresholds',
+    )
+
+    def __init__(self, pool, cr):
+        """ Override to check if env var to hide threshold configuration and
+        reset the database state is set. If it is, run those actions
+        """
+        if HIDE_THRESHOLD:
+            exempt_users = os.environ.get(
+                'USER_THRESHOLD_USER', ''
+            ).split(',')
+            cr.execute(
```

Comment:
No, I just had to create an environment, since `self.env` is not yet available in the `__init__` and the `pool` is weird. I went ahead and just generated a cursor and environment.